Changes to the book content: (#103)
ferrous-systems/espressif-trainings: book/src/01_intro.md

<p style="text-align:center;"><img src="./assets/esp-logo-black.svg" width="50%"></p>

# Introduction

## Content of this material

This is Ferrous Systems' *Embedded Rust on Espressif* training material. It is divided into two workshops: introductory and advanced. The introductory trail will introduce you to basics of embedded development and how to make the embedded board interact with the outside world - reacting to commands and sending sensor data.

The advanced course takes it from there to dive deeper into topics like interrupt handling, low-level peripheral access and writing your own drivers.

> ⚠️ You will be able to chat with our experts during the training if you are participating to the training. In addition, we strongly suggest to join the [community](https://matrix.to/#/#esp-rs:matrix.org) on the Matrix for all technical questions or delivery issues! The community is open to everyone.

Reviewer: ```suggestion
You can join the [esp-rs community](https://matrix.to/#/#esp-rs:matrix.org) on Matrix for all technical questions and  issues! The community is open to everyone.
```
The sentence contained too much information that is only relevant to this training.
